src/scripts/index.js

```diff
@@ -92,10 +92,11 @@
   function addBackToTop() {
     var link = doc.createElement('a');
     var container = doc.querySelector('body');
+    var text = options['back-to-top-text'] || 'Back to top';
```

Comment:
Could you add the default text to `theme.json` and remove the conditional here? that way you don't even need this variable.